public_html/wp-content/themes/wporg-events-2023/patterns/events-list-filters.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Title: Events List Filters
+ * Slug: wporg-events-2023/event-list-filters
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|30","bottom":"var:preset|spacing|30"}}},"layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignwide" style="margin-top:var(--wp--preset--spacing--30);margin-bottom:var(--wp--preset--spacing--30)"><!-- wp:group {"layout":{"type":"flex","flexWrap":"wrap"}} -->
+<div class="wp-block-group"><!-- wp:search {"showLabel":false,"placeholder":"Search events...","width":100,"widthUnit":"%","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true,"className":"is-style-secondary-search-control"} /--></div>
+<!-- /wp:group -->
+
+<!-- wp:template-part {"slug":"event-filters"} /-->
```

Comment:
I don't see this either, do you have it locally?

Reply:
I added it in 303e77f3 so I can work on it today, but free free to change that if you had something else in mind